weblate_schemas/schemas/weblate-component.schema.json

```diff
@@ -84,6 +84,15 @@
             "hello"
           ]
         },
+        "category": {
+          "$id": "#root/component/category",
+          "title": "Category slug",
```

Comment:
I don't think category slug is enough here. The categories can be nested and can have the same slug. For example, `project / my-category / my-category / my-category / component` is a valid structure.

Reply:
I see, full slug then?

Reply:
Yes, that would work well.